testing/KeyboardReport.cpp

```diff
@@ -34,8 +34,19 @@ std::vector<uint8_t> KeyboardReport::ActiveKeycodes() const {
     uint8_t key_code = report_data_.keys[i / 8] & bit;
     if (key_code) active_keys.push_back(i);
   }
+  for (uint8_t i{0}; i < 8; ++i) {
```

Comment:
Instead if the 8, could you calculate it with HID_KEYBOARD_LAST_MODIFIER?

Reply:
I thought about that, but really the right thing is "the number of bits in `report_data_.modifiers`". I'll change it to leave out the literal later tonight.